.eslintrc.js

```diff
@@ -1,15 +1,31 @@
 module.exports = {
-  "extends": [
-    "react-app",
-    "prettier/@typescript-eslint",
+  env: {
```

Comment:
Are these just copied from the protocol repo?

Reply:
it mostly was, i didnt need to add overrides and removed one rule about flattening brackets which was causing warnings.